packages/ios-client/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -715,6 +717,51 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
             break
         }
     }
+
+    func onDisconnected() {
+        DispatchQueue.webRTC.sync {
+            peerConnectionManager.close()
+            prevLocalTracks = Array((prevLocalTracks + localTracks))
```

Comment:
What would be the condition that `onDisconnect` is called with non empty `prevLocalTracks`?